pywbemtools/pywbemcli/pywbemcli.py

```diff
@@ -104,16 +106,16 @@
               help='Password for the WBEM server. '
                    'Default: EnvVar {ev}, or prompted for if --user specified.'.
                    format(ev=PywbemServer.password_envvar))
-@click.option('-N', '--no-verify', is_flag=True,
+@click.option('--verify/--no-verify', 'verify', default=None,
               # defaulted in code
-              envvar=PywbemServer.no_verify_envvar,
-              help='If true, client does not verify the X.509 server '
+              envvar=PywbemServer.verify_envvar,
+              help='If --verify, client verifies the X.509 server '
                    'certificate presented by the WBEM server during TLS/SSL '
-                   'handshake. '
-                   'Default: EnvVar {ev}, or false.'.
-                   format(ev=PywbemServer.no_verify_envvar))
+                   'handshake. If --no-verify client bypasses verification'
```

Comment:
Add "`. `" (dot and space) at the end of this line, to separate next line.

Reply:
DONE